e2e-tests/run-all

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -o xtrace
+set -o errexit
+
+dir=$(realpath "$(dirname "$0")")
+
+fail() {
```

Comment:
Do we plan to run all test only in [release pipeline](https://cloud.cd.percona.com/view/PXC/job/pxc-operator-gke-latest/)  not in PR? What do you think do we want to have an ability to run all test in PR? We can try to implement it in another PR.

Reply:
The initial idea was to run all tests in the release pipeline only. If we will have a need we can add it later.

Reply:
How would that look like, parametrized PR test?
I'm thinking that if we really have some PR for which the PR tests pass, but we have some concerns that it might break something in the tests which were skipped we can run our parametrized job manually to check it.